ols/app/endpoints/feedback.py

```diff
@@ -136,7 +140,8 @@ def store_user_feedback(
             detail={
                 "response": "Error storing user feedback",
                 "cause": str(e),
+                "timestamp": datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S'),
             },
         )
 
-    return FeedbackResponse(response="feedback received")
+    return FeedbackResponse(response="feedback received", timestamp=datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S'))
```

Comment:
I'd drop this there, it is not part of FeedbackResponse model

Reply:
okay so remove "timestamp=datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S'"